src/main.rs

```diff
@@ -14,6 +15,11 @@ fn main() {
     let cli_app = App::new(crate_name!())
         .setting(AppSettings::VersionlessSubcommands)
         .setting(AppSettings::SubcommandRequired)
+        .setting(if atty::is(Stream::Stdout) {
+            AppSettings::ColoredHelp
+        } else {
+            AppSettings::ColorNever // in case it run on terminal without color support
+        })
```

Comment:
Curious about this: for terminals without color support, what happens if we still activate `ColoredHelp`? Does it negatively affect those terminals in some way?

My actual question: is pulling in the `atty` dep actually worth it? Do we need to do this?

Reply:
<details><summary>It will output this gibberish</summary>

```
[32mmask[0m 0.5.0
Jake Deichert <git@jakedeichert.com>
A CLI task runner defined by a simple markdown file
[33m
USAGE:[0m
    mask [OPTIONS] <SUBCOMMAND>

[33mFLAGS:
[0m    [32m-h[0m, [32m--help[0m       Prints help information
    [32m-V[0m, [32m--version[0m    Prints version information

[33mOPTIONS:
[0m        [32m--maskfile[0m [32m<maskfile>[0m    Path to a different maskfile you want to use

[33mSUBCOMMANDS:
[0m    [32mbuild[0m     Build a release version of mask
    [32mdeps[0m      Commands related to cargo dependencies
    [32mformat[0m    Format all source files
    [32mhelp[0m      Prints this message or the help of the given subcommand(s)
    [32mlink[0m      Build mask and replace your globally installed version with it for testing
    [32mlint[0m      Lint the project with clippy
    [32mrun[0m       Build and run mask in development mode
    [32mtest[0m      Run all tests
```

You can simulate it by streaming _stdout_ directly to a file:
```sh
mask help > help.man
```
Then open `help.man` in the text editor.

---
</details>

However, seems I misunderstand about `ColoredHelp` setting. By default it will automatically disabled when running on non-TTY mode. It only output those gibberish if **both** `ColorAlways` and `ColoredHelp` enabled. I will remove `atty` check and deps shortly.

Reply:
done https://github.com/jakedeichert/mask/pull/30/commits/8c2c03b693cfba17b4ebf7e89d0273e0cc717b90

Reply:
done 8c2c03b

Reply:
@jakedeichert done 8c2c03b

Reply:
Thanks for the reminder, I’ll try this out tonight 👍